ptcg_cockatrice.xml

```diff
@@ -1691,7 +1691,7 @@ Whenever you evolve a Pokémon, Flareon deals 3 damage divided as you choose amo
 			<type>Pokémon - Rock Water</type>
 			<pt>2/3</pt>
 			<tablerow>2</tablerow>
-			<text>{1}{U}{U}: Omanyte can't be blocked this turn.
+			<text>{1}{U}{U}: Copy target instant spell with mana value 2 or less. You may choose new targets for the copy. 
```

Comment:
this is cool

Reply:
Wanted to give it an ability that still allowed Omastar to use its tap ability